src/mros2.cpp

```diff
@@ -225,7 +225,6 @@ Subscriber Node::create_subscription(std::string topic_name, int qos, void(*fp)(
   /* Register callback to ensure that a subscriber is matched to the reader before receiving messages */
   part_ptr->registerOnNewPublisherMatchedCallback(subMatch, &pubMatched);
 
-  MROS2_DEBUG("[MROS2LIB] create_subscription complete. data memory address=0x%x", data_p);
```

Comment:
suggested: 

>   MROS2_DEBUG("[MROS2LIB] create_subscription complete");

It is better to print some message at this point, but I agree that data memory address is not needed.

Reply:
thanks for suggestion. 
see:
 https://github.com/mROS-base/mros2/pull/24/commits/8961706dee733d1e825f55c311bbc4af9ce0cd5c